AlphaWallet/Tokens/Collectibles/ViewControllers/NFTAssetSelectionViewController.swift

```diff
@@ -315,7 +315,7 @@ extension NFTAssetSelectionViewController {
         isSearchBarConfigured = true
 
         if let placeholderLabel = searchController.searchBar.firstSubview(ofType: UILabel.self) {
-            placeholderLabel.textColor = Colors.lightGray
+            placeholderLabel.textColor = Configuration.Color.Semantic.searchbarPlaceholder
```

Comment:
"searchBar" with capital B too for consistency

Reply:
> "searchBar" with capital B too for consistency

That's taken care of in another set of PRs #6266 and #6265. @hboon

Reply:
@eviltofu no, right?